### PR TITLE
feat: 360 Image detail react component

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Feat  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/REV-910

## Description :pencil:
React component to support 360 Image details button for revision details, station name, station id

image

## How has this been tested? :mag:
In storybook & data exploration

## Test instructions :information_source:
1. `cd react-component && yarn build && yarn storybook`
2. Select `Image360HistoricalDetailsPanel` from storybook example

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/package.json [hunk 1]
```diff
@@ -14,13 +14,18 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build"
   },
+  "dependencies": {
+    "@cognite/cogs.js": "^9.13.2",
```
Comment (christjt) [inline react-components/package.json:18]: Should be peer dependency and dev dependency.

File: react-components/src/components/Panel/Image360HistoricalDetailsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { Chip, Tooltip } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+
+export interface Image360HistoricalDetailsPanelProps {
+  historicalCount?: number;
+  onDetailsClick?: () => void;
+}
+
+export const Image360HistoricalDetailsPanel = ({historicalCount, onDetailsClick}: Image360HistoricalDetailsPanelProps) => {
```
Comment (christjt) [inline react-components/src/components/Panel/Image360HistoricalDetailsPanel.tsx:14]: I assume this is just a placeholder. Eventually use a context to get a viewer instance right?

**Review by christjt — CHANGES_REQUESTED** [review]
Looks good as a starting point! Only a couple small comments / questions.

File: react-components/src/components/Panel/Image360HistoricalDetailsPanel.tsx (hunk 2 above)
Comment (pramodcog) [reply react-components/src/components/Panel/Image360HistoricalDetailsPanel.tsx:14]: Correct, these are placeholder 😄

**Review by christjt — APPROVED** [review]

**Pull request merged**